cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,48 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signupPageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+
+const signUpPage = new SignUpPageObject();
+const homePage = new HomePageObject();
+
 describe('Sign Up page', () => {
+  let user;
+  let newUser;
+
   before(() => {
 
   });
 
-  it('should ...', () => {
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
+    cy.task('generateUser').then((generatedUser) => {
+      newUser = generatedUser;
+    });
```

Comment:
Generating two users in the `beforeEach` hook might not be necessary for every test case. Consider whether both users are needed for each test or if this can be optimized.